beaker.gemspec

```diff
@@ -38,6 +38,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'net-ssh', '~> 2.9'
   s.add_runtime_dependency 'net-scp', '~> 1.2'
   s.add_runtime_dependency 'inifile', '~> 2.0'
+  s.add_runtime_dependency 'rsync'
```

Comment:
Please give an initial major version so that we don't get weird bustage on version bumps in external projects.